### DISCUSSION TITLE
Watchy PCF8563 (V1.5 -> 2.0) GSR users, show me your Drift numbers.

### DISCUSSION THREAD

**Discussion by GuruSR** [GuruSR/Watchy_GSR#16] [Q&A]
I'm asking this, you've got Watchy GSR running, have you done the Drift calculation yet?  If so, what number are you getting?

If you're unaware, Watchy GSR has a means of letting you manually set the start and set the time again at the finish of a (7+ day long drift test).

Read the information in [Edit RTC](https://github.com/GuruSR/Watchy_GSR/blob/main/Usage.md#user-content-fn-1-9d9680deb742a69e444838374019f770).

So, to help assist people with existing PCF8563's, I was hoping the drift numbers generated by users, would show up here to give me a  "sort of base line drift", so that way, PCF8563 users could be greeted with a "default drift" added based on the compilation from here, so if people can start giving their drift numbers here, it would be appreciated.  This way, people having the PCF8563 won't all be complaining that it is x minutes off after a day/week right away once using Watchy GSR.  It might be out by seconds or could be spot on.

**Comment by amartyaksinha-UC** [comment]
Hi! Thank you for opening discussions. 
I do not have a current drift number. However, can the drift problem be considered solved (or at least, subdued) if we keep the Sync Watchy to "On"? I am assuming the setting syncs time once a day at a set time, negating drift basically every day. If not, then do ignore this comment.

**Reply by GuruSR** [reply]
You can, but that defeats the purpose of keeping the Watchy's WiFi usage to a minimum, plus timers will not be as accurate as you would want them to be, the drift helps correct the time based on the noticed drift, so later on, when you want to time for 4 hours, you're not going to get 4 hours and 30 seconds (when whatever it was you were cooking goes over).

**Reply by amartyaksinha-UC** [reply]
Okay so my Watchy has not been synced for at least 5 days (likely about 7-8 days), and it is about 1min 30s ahead of the clock on my laptop. Assuming it's been 7 days, that gives us an average drift of +13s per day I suppose. Not sure if this was a linear increase over time or not, but I noticed it was about a minute ahead roughly 3-5 days after last sync so seems to be so far. I am going to sync my Watchy today and keep detailed notes on the drift this time round :)

**Reply by GuruSR** [reply]
The External RTC chips typically have some sort of temperature management for drift, but the ESP32 (at least in the 1.0 to 2.0 Watchys) doesn't, so thermal variance is going to happen.  The Drift code in SmallRTC 2.3 and using Watchy GSR 1.4.7C, the drift code is limited to 2 decimal math, so there could be some minor offset, but Watchy GSR 1.4.7C offers you the choice to tweak the Drift Value until you get it as close as possible.  This would reduce the WiFi usage and almost instantly remove that 1min 30 seconds of drift.  I tested my internal RTC with the new drift code, 7+ minutes in 1 day, now is roughly around 2 to 3 seconds a day, making the Internal RTC actually possible for usage for those with broken External RTC.s
